int8                 task_id
int8                 allocated_robot_id
bool                 known
bool                 completed
geometry_msgs/Point  point
int8                 order
